apollo-compiler/src/test/kotlin/com/apollographql/apollo3/compiler/ValidationTest.kt

```diff
@@ -45,6 +44,8 @@ class ValidationTest(name: String, private val graphQLFile: File) {
         val schemaResult = parseResult.valueAssertNoErrors().validateAsSchemaAndAddApolloDefinition()
         schemaResult.issues + if (graphQLFile.name == "reserved-enum-value-names.graphql") {
           checkApolloReservedEnumValueNames(schemaResult.value!!)
+        } else if (graphQLFile.name == "duplicate-target-name.graphql") {
+          checkApolloDuplicateTargetNames(schemaResult.value!!)
         } else {
```

Comment:
Sounds like the apollo-specific validation could apply for all schemas, irrespective of their name?